tasks/provision.yml

```diff
@@ -54,6 +54,7 @@
   mr_provisioner_machine_provision:
     machine_name: "{{ inventory_hostname }}"
     kernel_description: "{{ mr_provisioner_kernel_description }}"
+    kernel_options: "{{ mr_provisioner_kernel_options }}"
```

Comment:
Since this is an optional parameter, what happens if it is not set? I think it needs a default (like, see line 43 above).